frontend/services/gui/src/GUI_Service.cpp

```diff
@@ -387,12 +377,12 @@ bool GUI_Service::gui_rendering_execution(void* void_ptr,
     unsigned int fbo_color_buffer_gl_handle = 0;
     size_t fbo_width = 1;
     size_t fbo_height = 1;
-    gui_ptr->GetFBODataGL(fbo_color_buffer_gl_handle, fbo_width, fbo_height);
+    gui_ptr->GetFBOData_GL(fbo_color_buffer_gl_handle, fbo_width, fbo_height);
 
     result_image =
         megamol::frontend_resources::wrap_image({fbo_width, fbo_height}, fbo_color_buffer_gl_handle, channels);
 #else
-    auto fbo = gui_ptr->getFBOHandle();
+    auto fbo = gui_ptr->GetFBOData_CPU();
     result_image =
         megamol::frontend_resources::wrap_image({fbo->getWidth(), fbo->getHeight()}, fbo->colorBuffer, channels);
```

Comment:
If GetFBOData() would either return an `ImageWrapper` the distinction between `_GL` and `_CPU` could be removed. Or instead of `fbo_color_buffer_gl_handle/fbo->colorBuffer` one could return a generic type that can be wrapped by the ImageWrapper.

Reply:
Good idea, see commit 2b4ed5d